DetectDynamicJS.py

```diff
@@ -77,6 +77,11 @@ def doPassiveScan(self, baseRequestResponse):
         statusCode = responseInfo.getStatusCode()
         if statusCode != 200:
           return
+        
+        # Check if the script is world readable
```

Comment:
Not sure why you added this check. In which case would that change something? Scripts can always be included x-domain.